src/node.h

```diff
@@ -574,6 +570,24 @@ NODE_EXTERN async_context EmitAsyncInit(v8::Isolate* isolate,
 NODE_EXTERN void EmitAsyncDestroy(v8::Isolate* isolate,
                                   async_context asyncContext);
 
+class InternalCallbackScope;
+
+class CallbackScope {
```

Comment:
This is a newly exposed public class correct? If so it'd be great if there can be some documentation around how to use it IMO.